leptonai/templates/llm_by_lepton/README.md

```diff
@@ -13,6 +13,70 @@ Set the following environmental variables.
 * `TUNA_STREAM_CB_STEP`: (Optional) in streaming mode, the minimum number of tokens to generate in each new chunk. Smaller numbers send generated results sooner, but may lead to a slightly higher network overhead. Default value set to 3. Unless you are hyper-tuning for benchmarks, you can leave this value as default.
 * `MEDUSA`: (Optional) Run the inference with a pre-trained [Medusa](https://arxiv.org/abs/2401.10774) speculative decoding model. For example, to speed up llama2-70b, use `leptonai/Llama-2-70b-chat-4-heads`. We have provided a set of pre-trained medusa heads on [HuggingFace](https://huggingface.co/leptonai). To train medusa heads on your own model or with your own data, please [talk to us](mailto:info@lepton.ai).
 * `LORAS`: (Optional) We support running LORA models (also known as PEFT models) in the same deployment as the main model, by setting LORAS in a format `model_id:model_name`. If you have multiple models, separate them by `|`. For example, [therealcyberlord/llama2-qlora-finetuned-medical](https://huggingface.co/therealcyberlord/llama2-qlora-finetuned-medical) is based on `meta-llama/Llama-2-7b-chat-hf`, and you can specify `therealcyberlord/llama2-qlora-finetuned-medical:medical` to run it. On the client side, invoke it with `model=medical`.
+* `LEPTON_CHAT_TEMPLATE`: (Optional) We support selecting a chat template automatically based on the model name. If you want to set a specific chat template, you can set this variable based on the following chat template names:
```

Comment:
nit: is it possible for us to use all lower-case chat template name, and always use dash instead of underscore? AKA, in the backend, maybe we can always do `template_name.lower().replace("_", "-")` and then do matching. Right now it is a little bit hard for users to remember the upper or lower case, for example, `DBRX` is upper case, while most others are lower case. `chatglm_chatml` is underscore while `llama-2` is dash. It might be a little bit confusing.

Reply:
Done at https://github.com/leptonai/l3m/pull/639 😉